pages/book/statements.mdx

```diff
@@ -1,122 +1,174 @@
-- `-`, `+` - arithmetic operations, defined only for `Int` type
-- `!=`, `==` - equality operations
-- `>`, `<`, `>=`, `<=` - compare operations, defined only for the `Int` type
-- `&&`, `||` - logical `AND` and `OR`
+```tact
+{ // <- start of the block
+    // arbitrary statements:
+    let value: Int = 2 + 2;
+    dump(value);
+} // <- end of the block
 
-## Loops
+{ dump(2 + 2); } // a block with only one statement
+```
 
-### Repeat loop
+## Expression
 
-The Repeat Loop executes a block of code a specified number of times. In the example you provided, the code inside the loop will be executed 10 times.
+An expression statement is an expression used in a place where a statement is expected. The expression is evaluated and its result is discarded — therefore, it makes sense only for expressions that have side effects, such as executing a function or updating a variable.
 
 ```tact
-let a: Int = 1;
-repeat(10) {
-  a = a * a;
-}
+dump(2 + 2); // stdlib function
 ```
 
-> **Note**
-> Repeat number must be a 32-bit int or of range exception is thrown. Negative values are ignored.
+## Assignment
 
-### While loop
-
-The While Loop continues executing the block of code as long as the given condition is true.
+Assignment statements use an [assignment operator](/book/operators#assignment) (`={:tact}`) or [augmented assignment operators](/book/operators#augmented-assignment) (assignments combined with an operation):
 
 ```tact
-let x: Int = 10;
-while(x > 0) {
-  x = x - 1;
-}
+let value: Int; // declaration
+value = 5;      // assignment
+value += 5;     // augmented assignment (one of the many, see below)
 ```
 
-**Explanation**:
+<Callout>
 
-- The code inside the loop will execute as long as � > 0 x>0.
-- In the example, the value of � x is decremented by 1 in each iteration, so the loop will run 10 times.
+  Read more about assignment and augmented assignment in their dedicated section: [assignment operators](/book/operators#assignment).
 
-### Until loop
+</Callout>
 
-The Until Loop is a post-test loop that executes the block of code at least once and then continues to execute it until the given condition becomes true.
+## Branches
 
-```tact
-let x: Int = 10;
-do {
-  x = x - 1;  # do something no matter at least one time
-} until (x <= 0);
-```
+Control the flow of the code.
+
+### `if...else` [#if-else]
 
-**Explanation**:
+<Callout type="warning" emoji="⚠️">
 
-- The code inside the loop will execute at least once, and then it will continue to execute until \( x \leq 0 \).
-- In the example, the value of \( x \) is decremented by 1 in each iteration, so the loop will run 10 times.
+  Curly brackets (code blocks) are required!
 
-## If Statements
+</Callout>
 
-> **Warn**
-> Curly brackets are required
+The `if...else{:tact}` statement executes a statement if a specified condition is `true{:tact}`. If the condition is `false{:tact}`, another statement in the optional `else{:tact}` clause will be executed.
```

Comment:
Let me commit this suggestion and merge the PR, because due to its size it's a bit hard to review updates to it

Reply:
oops, I somehow cannot commit it

Reply:
Huh, I don't think your write-up here makes it clearer. Because after my rather short description goes thorough set of code examples, which clarify the rest